pkg/dockerregistry/server/manifestservice.go

```diff
@@ -143,6 +148,7 @@ func (m *manifestService) Put(ctx context.Context, manifest distribution.Manifes
 			DockerImageReference:         fmt.Sprintf("%s/%s/%s@%s", m.repo.config.registryAddr, m.repo.namespace, m.repo.name, dgst.String()),
 			DockerImageManifest:          string(payload),
 			DockerImageManifestMediaType: mediaType,
+			DockerImageConfig:            string(config),
```

Comment:
It will land in etcd, won't it?

Reply:
> It will land in etcd, won't it?

I've updated the image strategy to unset `DockerImageConfig` and `DockerImageManifest`.